doc/source/release/1.16.0-notes.rst

```diff
@@ -41,10 +41,13 @@ New features
 ``scipy.differentiate`` improvements
 ====================================
 
-
 ``scipy.interpolate`` improvements
 ==================================
+- Improved the error estimate of `scipy.integrate.tanhsinh`
 
+``scipy.interpolate`` improvements
+==================================
+- Added batch support to `scipy.interpolate.make_smoothing_spline`
```

Comment:
@ev-br were there other splines that got batch support, or was there anything else you wanted to add to the release notes? Feel free to add specific wording as a suggestion here (that can just be committed), or if this is merged first, please add to scipy/scipy#22991.

Reply:
Doesn't this duplicate the `scipy.interpolate` heading from just above? I think I'll probably just merge this PR in a minute after looking it over in any case though, and just carry that comment over.